vagrant/provisioning/roles/activemq/tasks/main.yml

```diff
@@ -70,7 +70,7 @@
   replace:
     path: "{{ root_folder }}/app/activemq/bin/linux-x86-64/wrapper.conf"
     regexp: "^wrapper.java.command=java"
-    replace: "wrapper.java.command={{ activemq_java_home | default('/usr/lib/jvm/java-11-openjdk') }}/bin/java"
+    replace: "wrapper.java.command={{ activemq_java_home | default('/usr/lib/jvm/amazon-corretto-11.0.27.6.1-linux-x64') }}/bin/java"
```

Comment:
Is there a more generic path, like `/usr/lib/jvm/amazon-corretto-11-linux-x64`?  or only this path with the specific version?